apps/livreGUI/Controller.cpp

```diff
@@ -21,238 +21,102 @@
  */
```

Comment:
If you think about registering the same ZeroBuf object to multiple subscribers that 'feature' does not make sense. In the end the event today has the same semantics. If multiple classes/widgets need an update/notification on a change of an event/object, this should be done in the application and not in the subscriber. Especially in the Qt world, you would signal that change anyways to connected slots from interested objects for proper update handling. Right now, every event is dedicated to one widget/GUI control, so there is no need for that.

The implementation now uses the shared subscriber feature, but this only 'helps' for a having a single receive which is not really critical. I'm not sure to what else you're referring to.